Formula/o/openstackclient.rb

```diff
@@ -30,284 +30,399 @@ class Openstackclient < Formula
-  end
-
-  resource "python-keystoneclient" do
+end
+            
+resource "python-keystoneclient" do
     url "https://files.pythonhosted.org/packages/13/c3/456c3d3125a189adf7d23e7ca7759226d717fdce20c6c590686d03886897/python-keystoneclient-5.2.0.tar.gz"
     sha256 "72a42c3869e2128bb0c626ac856c3dbf3e38ef16d7e85dd35567b82cd24539a9"
-  end
-
-  resource "python-neutronclient" do
+end
+            
+resource "python-magnumclient" do
+    url "https://files.pythonhosted.org/packages/d5/a7/a0da4839ef91babf4166eec2b395f85737c0b816816f3ab3970a528f9a59/python-magnumclient-4.3.0.tar.gz"
+    sha256 "b6cafb1069a0ec6a636b89639e82cbe6bc6903fb1ccdd583f9c3fd868d275b34"
+end
+            
+resource "python-manilaclient" do
+    url "https://files.pythonhosted.org/packages/a4/60/b9143015703487519c6bbd8ae70e6ca84a9d42361c211d9f620d4428d150/python-manilaclient-4.7.0.tar.gz"
+    sha256 "6e5904fa92f3650f0108702693ac8d8f3a9b04094ad9a6f5172486141fe43767"
+end
+            
+resource "python-mistralclient" do
+    url "https://files.pythonhosted.org/packages/27/6e/18c53ae5a3766e7872ad2732971ea35fd2e40a4bc91fbd71a128dfce0da0/python-mistralclient-5.1.0.tar.gz"
+    sha256 "104d1e93404bc90c430ec8ad1ba9a333224d5cf9684bac702ca724560fbe9739"
+end
+            
+resource "python-monascaclient" do
+    url "https://files.pythonhosted.org/packages/42/64/2e668680115c24499cdc5fba54022f154aa1760b4a2c5fe4c21d27dd00d3/python-monascaclient-2.8.0.tar.gz"
+    sha256 "efe223d59a6bf2e0f24e4af70be4d2382a3cec4076fa7f40a5aa70adf05a12d2"
+end
+            
+resource "python-muranoclient" do
+    url "https://files.pythonhosted.org/packages/6e/90/d73d6be423c097021d7c54a74a3640120ad06dad1a8fd5718ce99861059b/python-muranoclient-2.7.0.tar.gz"
+    sha256 "df96c5ded0ba0299ca0fe38e3f5775c889c03923fc8d7e94f6b96dbf679e19d6"
+end
+            
+resource "python-neutronclient" do
     url "https://files.pythonhosted.org/packages/1e/2d/c58cc8c8831fc433d415f3d30f5a84a5da0ade1e16e20fee962d4e4fa7b8/python-neutronclient-11.1.0.tar.gz"
     sha256 "f0a5cd6e3d8ce5a26fb0f88a932148a7a304148f6d8c68bc58e9ba1cc38d7798"
-  end
-
-  resource "python-novaclient" do
+end
+            
+resource "python-novaclient" do
     url "https://files.pythonhosted.org/packages/c4/c6/4ee6c69bfc7a9e9dde92d946594735f638dcf4959e6e8da1b1d920b3f0d9/python-novaclient-18.4.0.tar.gz"
     sha256 "6b6b6ae2c11eb1c108e3af55eaa7211b0fc9199935a229a6ba3e0de514c12b50"
-  end
-
-  resource "python-octaviaclient" do
-    url "https://files.pythonhosted.org/packages/ed/2d/5bfd38460b8857c0485075fac757d6546a4c080d003934a38c802465a07a/python-octaviaclient-3.6.0.tar.gz"
-    sha256 "96488ab87bb4566f2d36a4a17eff97d57d9875e0966a9ae296aeaafbc23788d2"
-  end
-
-  resource "python-swiftclient" do
+end
+            
+resource "python-openstackclient" do
+    url "https://files.pythonhosted.org/packages/71/13/9556dd8c47b78a7542f5f9241f4798f66d1203b6fd5f069518f5cc51df3f/python-openstackclient-6.4.0.tar.gz"
```

Comment:
this shouldn't have been added since this duplicates the formula's url, can you run `brew update-python-resources openstackclient` again?

Reply:
~/homebrew-core> brew update-python-resources openstackclient 
```
==> Retrieving PyPI dependencies for "python-openstackclient==6.4.0 python-designateclient python-heatclient python-octavi
==> Retrieving PyPI dependencies for excluded "certifi cffi cryptography packaging pycparser pyparsing pyyaml six pytz"...
==> Getting PyPI info for "attrs==23.2.0"
==> Getting PyPI info for "autopage==0.5.2"
==> Excluding "certifi==2023.11.17"
==> Excluding "cffi==1.16.0"
==> Getting PyPI info for "charset-normalizer==3.3.2"
==> Getting PyPI info for "cliff==4.4.0"
==> Getting PyPI info for "cmd2==2.4.3"
==> Excluding "cryptography==41.0.7"
==> Getting PyPI info for "debtcollector==2.5.0"
==> Getting PyPI info for "decorator==5.1.1"
==> Getting PyPI info for "dogpile-cache==1.3.0"
==> Getting PyPI info for "idna==3.6"
==> Getting PyPI info for "importlib-metadata==7.0.1"
==> Getting PyPI info for "iso8601==2.1.0"
==> Getting PyPI info for "jmespath==1.0.1"
==> Getting PyPI info for "jsonpatch==1.33"
==> Getting PyPI info for "jsonpointer==2.4"
==> Getting PyPI info for "jsonschema==4.20.0"
==> Getting PyPI info for "jsonschema-specifications==2023.12.1"
==> Getting PyPI info for "keystoneauth1==5.4.0"
==> Getting PyPI info for "msgpack==1.0.7"
==> Getting PyPI info for "netaddr==0.10.1"
==> Getting PyPI info for "netifaces==0.11.0"
==> Getting PyPI info for "openstacksdk==2.0.0"
==> Getting PyPI info for "os-client-config==2.1.0"
==> Getting PyPI info for "os-service-types==1.7.0"
==> Getting PyPI info for "osc-lib==2.9.0"
==> Getting PyPI info for "oslo-config==9.3.0"
==> Getting PyPI info for "oslo-context==5.3.0"
==> Getting PyPI info for "oslo-i18n==6.2.0"
==> Getting PyPI info for "oslo-log==5.4.0"
==> Getting PyPI info for "oslo-serialization==5.3.0"
==> Getting PyPI info for "oslo-utils==6.3.0"
==> Excluding "packaging==23.2"
==> Getting PyPI info for "pbr==6.0.0"
==> Getting PyPI info for "platformdirs==4.1.0"
==> Getting PyPI info for "prettytable==3.9.0"
==> Excluding "pycparser==2.21"
==> Excluding "pyparsing==3.1.1"
==> Getting PyPI info for "pyperclip==1.8.2"
==> Getting PyPI info for "python-cinderclient==9.4.0"
==> Getting PyPI info for "python-dateutil==2.8.2"
==> Getting PyPI info for "python-designateclient==5.3.0"
==> Getting PyPI info for "python-heatclient==3.4.0"
==> Getting PyPI info for "python-keystoneclient==5.2.0"
==> Getting PyPI info for "python-neutronclient==11.1.0"
==> Getting PyPI info for "python-novaclient==18.4.0"
==> Getting PyPI info for "python-octaviaclient==3.6.0"
==> Excluding "python-openstackclient==6.4.0"
==> Getting PyPI info for "python-swiftclient==4.4.0"
==> Excluding "pytz==2023.3.post1"
==> Excluding "pyyaml==6.0.1"
==> Getting PyPI info for "referencing==0.32.1"
==> Getting PyPI info for "requests==2.31.0"
==> Getting PyPI info for "requestsexceptions==1.4.0"
==> Getting PyPI info for "rfc3986==2.0.0"
==> Getting PyPI info for "rpds-py==0.16.2"
==> Getting PyPI info for "simplejson==3.19.2"
==> Excluding "six==1.16.0"
==> Getting PyPI info for "stevedore==5.1.0"
==> Getting PyPI info for "tzdata==2023.4"
==> Getting PyPI info for "urllib3==2.1.0"
==> Getting PyPI info for "wcwidth==0.2.13"
==> Getting PyPI info for "wrapt==1.16.0"
==> Getting PyPI info for "zipp==3.17.0"
==> Updating resource blocks
```
 ~/homebrew-core> brew style --fix openstackclient
```

1 file inspected, no offenses detected
```
 ~/homebrew-core> git status       
```                                         
On branch openstack-cli-upgrade
Your branch is up to date with 'origin/openstack-cli-upgrade'.

nothing to commit, working tree clean
```

Reply:
Found the issue:
`python-troveclient` ""requires"" python-openstackclient in its dependencies. Thus creating the entry in the list of depens.

``` pip3 show python-troveclient                                   
Name: python-troveclient
Version: 8.2.1
Summary: Client library for OpenStack DBaaS API
Home-page: https://docs.openstack.org/python-troveclient/latest
Author: OpenStack
Author-email: openstack-discuss@lists.openstack.org
License: UNKNOWN
Location: /opt/homebrew/lib/python3.11/site-packages
Requires: keystoneauth1, osc-lib, oslo.i18n, oslo.utils, pbr, PrettyTable, python-mistralclient, python-openstackclient, python-swiftclient, requests, simplejson
```

Reply:
@branchvincent